pkg/loki/query.go

```diff
@@ -140,6 +141,21 @@ func (q *Query) WriteLabelFilter(sb *strings.Builder, lfs *[]labelFilter, lj Lab
 	}
 }
 
+func (q *Query) AddParams(params url.Values) error {
+	for key, values := range params {
+		if len(values) == 0 {
+			// Silently ignore
+			return nil
```

Comment:
shouldn't it be a `continue` to keep looking for the other params?

Reply:
good catch! thanks